app/templates/about.jade

```diff
@@ -1,188 +1,380 @@
+          h2 Our mission: make programming accessible to every student on Earth.
+          #mission-graphic-filler
+        .col-sm-5.col-sm-offset-1
+          p#mission-text.responsive-side-margins
+            span.spr Programming is magic. It's the ability to create things from pure imagination. We started CodeCombat to give learners the feeling of wizardly power at their fingertips, straight from the first keystroke. With game-based learning, players not only get more excited about computer science, but they learn faster, too. Way faster. It's like having a conversation instead of reading a book. We want to bring that conversation to every school and to every student, because we believe that everyone should have a chance to become a native speaker of code.
+
+      #team.anchor
+        h3 Meet the CodeCombat team
+        h4.responsive-side-margins We value open and respectful dialog, where the best idea wins. Our decisions are grounded in customer research and our process is focused on delivering tangible results for them. Everyone is hands-on, from our CEO to our Github contributors, because we value growth and learning in our team.
+        ul
           // Full time
-
-          a(href="http://www.nickwinter.net")
-            img(src="/images/pages/about/nick_small.png").img-thumbnail
-          .team_bio
-            h4.team_name
-              a(href="http://www.nickwinter.net") Nick Winter
-            div(data-i18n="about.nick_title")
-            div(data-i18n="about.nick_blurb")
-
-          a(href="http://www.mattlott.com/")
-            img(src="/images/pages/about/matt_small.png").img-thumbnail
-          .team_bio
-            h4.team_name
-              a(href="http://www.mattlott.com/") Matt Lott
-            div(data-i18n="about.matt_title")
-            div(data-i18n="about.matt_blurb")
-
-        li.row
-
-          a(href="http://cat.zdh.com/")
-            img(src="/images/pages/about/cat_small.png").img-thumbnail
-          .team_bio
-            h4.team_name
-              a(href="http://cat.zdh.com/") Catherine Weresow
-            div(data-i18n="about.cat_title")
-            div(data-i18n="about.cat_blurb")
-
-          img(src="/images/pages/about/scott_small.png").img-thumbnail
-          .team_bio
-            h4.team_name Scott Erickson
-            div(data-i18n="about.scott_title")
-            div(data-i18n="about.scott_blurb")
-
-        li.row
-
-          img(src="/images/pages/about/maka_small.png").img-thumbnail
-          .team_bio
-            h4.team_name Michael 'Maka' Gradin
-            div(data-i18n="about.maka_title")
-            div(data-i18n="about.maka_blurb")
-
-          a(href="http://basicer.com/")
+          li
+            a(href="http://www.nickwinter.net")
+              img(src="/images/pages/about/nick_small.png").img-thumbnail
+            .team_bio
+              h6.label.team_name
+                a(href="http://www.nickwinter.net") Nick Winter
```

Comment:
CSS classes and ids should use hyphens instead of underscores. Looks like these are legacy... good opportunity to standardize them!